src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -0,0 +1,356 @@
+
+    align-self: center;
+    width: var(--pf-c-data-toolbar--m-separator--Width);
+    height: var(--pf-c-data-toolbar--m-separator--Height);
+    background-color: var(--pf-c-data-toolbar--m-separator--BackgroundColor);
+  }
+
+  // Group
+  .pf-m-button-group {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--m-button-group--spacer);
+
+    > * {
+      --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--m-button-group--space-items);
+    }
+  }
+
+  // Icon button group
+  .pf-m-icon-button-group {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--m-icon-button-group--spacer);
+
+    > * {
+      --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--m-icon-button-group--space-items);
+    }
+  }
+
+  // Filter group
+  .pf-m-filter-group {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--m-filter-group--spacer);
+
+    > * {
+      --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--m-filter-group--space-items);
+    }
+  }
+
+  // Search filter
+  .pf-m-search-filter {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-search-filter--spacer);
+
+    > * {
+      --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-search-filter--space-items);
+    }
+  }
+
+  // Overflow menu
+  .pf-m-overflow-menu > * {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-overflow-menu--space-items);
+  }
+
+  // Bulk select
+  .pf-m-bulk-select {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-bulk-select--spacer);
+  }
+
+  // Label
+  .pf-m-label {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-label--spacer);
+
+    font-weight: var(--pf-c-data-toolbar--m-label--FontWeight);
+  }
+
+  // Total items
+  .pf-m-pagination {
+    margin-left: auto;
+
+    .pf-c-pagination {
+      flex-wrap: nowrap;
+    }
+  }
+
+  .pf-m-clear {
+    justify-self: end;
+  }
+
+  // If pagination is sibling of align-right
+  .pf-m-align-right ~ * {
+    margin-left: 0;
+  }
+
+  .pf-c-chip-group {
+    align-items: baseline;
+  }
+}
+
+.pf-c-data-toolbar__content,
+.pf-c-data-toolbar__group {
+  display: flex;
+  align-items: center;
+}
+
+// Set all spacer values to '--pf-c-data-toolbar--spacer' in order to change this variable value contextually
+.pf-c-data-toolbar__content {
+  padding: var(--pf-c-data-toolbar__content--PaddingTop) var(--pf-c-data-toolbar__content--PaddingRight) var(--pf-c-data-toolbar__content--PaddingBottom) var(--pf-c-data-toolbar__content--PaddingLeft);
+
+  & + & {
+    padding-top: 0;
+  }
+}
+
+// Group
+.pf-c-data-toolbar__group {
+  --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--spacer);
+}
+
+// Item
+.pf-c-data-toolbar__item {
+  // reset spacer
+  --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__item--spacer);
+}
+
+.pf-c-data-toolbar__group,
+.pf-c-data-toolbar__item {
+  margin-right: var(--pf-c-data-toolbar--spacer);
+
+  &:last-child:not([class*="pf-m-spacer"]) {
```

Comment:
nice! probably best to allow them to define this spacing even if it's the last child.